src/pages/myNotes/noteListItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import { StyledSpan } from "./styled";
+
+const NoteListItem = ({
+  getItemInfo,
+  id,
+  title,
+  description,
+  date,
+  showId,
+}) => {
+  return (
+    <div
+      className="notes-list__component-text"
+      onClick={getItemInfo ? getItemInfo : null}
+      id={id ? id : null}
+    >
+      <h2>{title}</h2>
+      {showId ? (
```

Comment:
change to `showId && {}`, ternary operator is redundant

Reply:
Changed